terraform/modules/monitoring/prober.tf

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+resource "google_cloud_run_v2_job" "pmap_prober" {
+
+  project = var.project_id
+
+  name = "pmap-prober"
+
+  location = "us-central1"
+
+  template {
+
+    template {
+
+      max_retries = 3
+
+      containers {
+        image = var.pmap_prober_image
+
+        env {
+          name  = "PROBER_PROJECT_ID"
+          value = var.project_id
+        }
+
+        env {
+          name  = "PROBER_BUCKET_ID"
+          value = var.prober_bucket_id
+        }
+
+        env {
+          name  = "PROBER_BIGQUERY_DATASET_ID"
+          value = var.prober_bigquery_dataset_id
+        }
+
+        env {
+          name  = "PROBER_MAPPING_TABLE_ID"
+          value = var.prober_mapping_table_id
+        }
+
+        env {
+          name  = "PROBER_POLICY_TABLE_ID"
+          value = var.prober_policy_table_id
+        }
+
+        env {
+          name  = "PROBER_QUERY_RETRY_WAIT_DURATION"
+          value = var.prober_query_retry_wait_duartion
+        }
+
+        env {
+          name  = "PROBER_QUERY_RETRY_COUNT"
+          value = var.prober_query_retry_count
+        }
+
+        env {
+          name  = "PROBER_MAPPING_GCS_BUCKET_PREFIX"
+          value = var.prober_mapping_gcs_bucket_prefix
+        }
+
+        env {
+          name  = "PROBER_POLICY_GCS_BUCKET_PREFIX"
+          value = var.prober_policy_gcs_bucket_prefix
+        }
+      }
+      service_account = resource.google_service_account.prober_service_account.email
```

Comment:
We usually save the `resource.` when referencing resources?

Reply:
Updated.